app/src/main/java/org/jellyfin/androidtv/details/EpisodeDetailsFragment.kt

```diff
@@ -8,9 +8,14 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
 import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.details.actions.BaseAction
+import org.jellyfin.androidtv.details.actions.PlayFromBeginningAction
+import org.jellyfin.androidtv.details.actions.ResumeAction
 import org.jellyfin.androidtv.model.itemtypes.Episode
 import org.jellyfin.apiclient.model.dto.ImageOptions
 
+private const val LOG_TAG = "EpisodeDetailsFragment"
+
 class EpisodeDetailsFragment(private val data: Episode) : BaseDetailsFragment() {
```

Comment:
I'm now thinking it might be nicer to use "episode" instead of "data". Looks more clear when using the actual data.

Reply:
Will do